.github/workflows/swift-toolchain.yml

```diff
@@ -1119,6 +1135,9 @@ jobs:
                 -D CMAKE_MT=mt `
                 -D CMAKE_INSTALL_PREFIX=${{ github.workspace }}/BuildRoot/Library/zlib-1.3/usr `
                 -D CMAKE_SYSTEM_NAME=${{ matrix.os }} `
+                -D CMAKE_ANDROID_NDK=$NDKPATH `
+                -D CMAKE_ANDROID_API=${{ needs.context.outputs.ANDROID_API_LEVEL }} `
+                -D CMAKE_POSITION_INDEPENDENT_CODE=${{ matrix.os == 'Android' && 'YES' || 'NO' }} `
```

Comment:
I think that we can set this unconditionally always, can't we?

Reply:
Yes, actually. Thanks for pointing that out